content/Watsonx/Assistant/104.md

```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
 
 The Analyze page displays high-level metrics on the performance of your assistant. These metrics and their underlying data are populated in real time. Take a moment to look around – you will see metrics such as number of unique users, conversations, and requests handled by the assistant in the specified date range.
 
+A **request** is a message from an end user. A request might be recognized, in which case the assistant would have used it to trigger an action. A request might also be unrecognized, in which case the assistant would have fallen back to disambiguation, escalation to a live agent, or asking the end user to try again.
+
+As you saw above, you can filter by environment. The Trial and Plus plans of watsonx Assistant offer two environments called Draft and Live. The Enterprise plan offers five environments, including Draft, Live, and up to three test environments, which more closely correlates to typical enterprise product lifecycle management situations, with their multiple staging areas.
+
+**Industry tips:**
+1. **Requests** are known as **utterances** in the field of conversational AI.
+2. The **Draft** and **Live** environments are analogous to **dev** and **prod** environments.
+
 ### Adding data to the Analyze page via Lendyr
 
-Open your assistant on the Lendyr using the link you created in the previous section to embed your assistant. Try out a few different conversations. To engage in an additional separate conversation with your assistant there should be a **Restart** button in upper-left menu.
+Conversation data is added to the Analytics page when your assistant engages in conversation with end users.
+
+Open your assistant on the Lendyr website (consult the **Preview your assistant on Lendyr Bank’s website** portion of section 2. Actions to open your assistant on the Lendyr website if you do not know how to do this) and try out a few conversations. To engage in an additional separate conversation with your assistant, you can restart the assistant by opening your custom Lendyr URL in a new tab or by following these instructions:
+
+1. Open a **Guided Tour** tile from the splash screen on your custom Lendyr.com website.
+
```

Comment:
Change guided tour to Accessible & Scalable AI. There is a quiz for this guided tour. Also, include an alert for the quiz in this section.